README – Linear Regression.md

```diff
@@ -0,0 +1,113 @@
+# Electric Car Battery Linear Regression Example
```

Comment:
This looks way to long , it looks a bit scary. We need to figure out how to shorten it. 

Also let's put table of content . You can use https://ecotrust-canada.github.io/markdown-toc/ to generate it quickly.